### DISCUSSION TITLE
Workflow level execution limit

### DISCUSSION THREAD

**Discussion by soumit09** [Netflix/conductor#2430] [General]
Hi Team, there is feature available on conductor to limit execution on task level. But i am looking for something limit execution on workflow level. My use case is same  down stream services can be us via multiple workflows and task level concurrentExecLimit can't help me as they are limited to same workflow executions.

**Comment by apanicker-nflx** [comment]
Currently, Conductor does not support execution limit at a workflow level. However, to meet your use-case, you could define a common task as the first step in your workflows and use the concurrentExecLimit on this task to control the number of workflow executions, as a workaround.
